frontend/src/components/explorer/gemBrowser/Reaction.vue

```diff
@@ -106,7 +106,7 @@ export default {
     }
 
     const title = `${this.reaction.id}, Reaction in ${this.model.short_name}`;
-    const description = `The reaction ${this.reaction.id} in ${this.model.short_name} (version ${this.model.version}) can be found in the ${this.reaction.compartments.[0].name} compartment and the ${this.reaction.subsystems[0].name} subsystem.`;
+    const description = `The reaction ${this.reaction.id} in ${this.model.short_name} (version ${this.model.version}) can be found in the ${this.reaction.compartments[0].name} compartment and the ${this.reaction.subsystems[0].name} subsystem.`;
```

Comment:
1. This feels like a random change to include in this PR
2. How would you suggest to test this change?

Reply:
1. It was reported by `logs frontend` so yeah pretty random.
2. The `description` field non-visible html. To test, I inspect the page and look for `description`.

Reply:
It is weird, I can't see any difference in the description when comparing between branch develop and this branch. But the recent changes do look more correct code wise.

However, I did notice though that for `/explore/Human-GEM/gem-browser/reaction/MAR12870`, the description is:

"The reaction MAR12870 in Human-GEM (version 1.8.0) can be found in the Extracellular compartment and the Drug metabolism subsystem."

I.e, Cytosol is not listed. I guess we should create an issue for this?

Would also like to add that I think that unrelated changes that require testing to be reviewed should be in a separate PR, with a description on how to test the changes

Reply:
To me, I corrected a typo in accordance with the recommendation from `logs frontend`. I didn't think this would change the code, or require review. I guess the lesson learnt is to open issues for typos.
Yes please a new issue for this.

Reply:
Makes sense! To me a typo that does not require a separate PR would be for example spelling errors, e.g. `The top table pf the page` to  `The top table of the page`, or fixing indentation. Changes that do not affect the logic of the code. This is a very small change but still it changes how properties of an object is accessed (right?). But yeah I would also have assumed that `logs frontend` would only report non functional changes.

Created issue #754 for the problem mentioned above

Reply:
Hmm I see your point. Here's how `logs frontend` looks without the fix (eg in `develop`):

```
frontend  |  ERROR  Failed to compile with 1 error4:18:56 PM
frontend  | 
frontend  |  error  in ./src/components/explorer/gemBrowser/Reaction.vue?vue&type=script&lang=js&
frontend  | 
frontend  | Syntax Error: Unexpected token (109:167)
frontend  | 
frontend  |   107 |
frontend  |   108 |     const title = `${this.reaction.id}, Reaction in ${this.model.short_name}`;
frontend  | > 109 |     const description = `The reaction ${this.reaction.id} in ${this.model.short_name} (version ${this.model.version}) can be found in the ${this.reaction.compartments.[0].name} compartment and the ${this.reaction.subsystems[0].name} subsystem.`;
frontend  |       |                                                                                                                                                                        ^
frontend  |   110 |
frontend  |   111 |     return {
frontend  |   112 |       title,
```

I do agree that it was surprising to see this in the review. Perhaps I should have included the error above in the PR. But yeah next time it would be more straightforward to fix this via a different issue.

Reply:
That's interesting, I don't get any problems when I run `logs frontend`. @mihai-sysbio you mentioned you did a clean install so maybe some packages are more updated than the rest of the project and it's picking up different errors?

Reply:
Could very well be. Just by looking at `this.reaction.compartments.[0].name` and `${this.reaction.subsystems[0].name` side by side still makes me think of a (code-)typo.